configs/BUILD

```diff
@@ -19,6 +19,7 @@ py_binary(
     data = glob([
         "*.yaml",
     ]),
+    python_version = "PY3",
```

Comment:
these were all removed long ago - nothing should be using the host python anymore - this not required here as it uses a `py_binary` which will automatically use the hermetic python

Reply:
> nothing should be using the host python anymore

in theory anyway - there are some non-hermetic bits still to be weeded out - one came to light testing removal of `PATH` from `action_env` (in a `configure_make`)

Reply:
Ok makes sense. Closing as WAD